meshtastic/tests/test_node_runtime_settings.py

```
@@ -0,0 +1,1309 @@
+    EMPTY_LONG_NAME_MSG,
+    EMPTY_SHORT_NAME_MSG,
+    MAX_LONG_NAME_LEN,
+    MAX_SHORT_NAME_LEN,
+)
+from ..protobuf import admin_pb2, config_pb2, localonly_pb2
+from ..util import Acknowledgment
+
+
+def _raise_test_error(msg: str) -> None:
+    """Raise an exception with the provided message."""
+    raise Exception(msg)
```

Ruff (0.15.6)

[warning] Create your own exception

(TRY002)